pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -75,1390 +66,988 @@
     SRPM_UNSIGNED_FIXTURE_PACKAGE_COUNT,
     SRPM_UNSIGNED_FIXTURE_URL,
 )
-from pulp_rpm.tests.functional.utils import (
-    gen_rpm_client,
-    gen_rpm_remote,
-    skip_if,
-)
+from pulp_rpm.tests.functional.utils import gen_rpm_remote
 from pulp_rpm.tests.functional.utils import set_up_module as setUpModule  # noqa:F401
 
-from pulpcore.client.pulp_rpm import (
-    ContentDistributionTreesApi,
-    ContentPackagesApi,
-    RepositoriesRpmApi,
-    RpmRepositorySyncURL,
-    RemotesRpmApi,
-    PublicationsRpmApi,
-)
+from pulpcore.client.pulp_rpm import RpmRepositorySyncURL
 from pulpcore.client.pulp_rpm.exceptions import ApiException
 
 
-class BasicSyncTestCase(PulpTestCase):
-    """Sync a repository with the rpm plugin."""
+@pytest.fixture
+def init_and_sync(gen_object_with_cleanup, monitor_task, rpm_repository_api, rpm_rpmremote_api):
```

Comment:
I would probably call this `basic_init_and_sync` or something, as other tests may need to "init and sync" another repo.

Reply:
Never mind, I see that you can actually pass the URL in, I missed the fact that this is a closure